playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -99,6 +127,8 @@
       env: "{{ env }}"
       host-type: "{{ host_type }}"
       env-host-type: "{{ env_host_type }}"
+      node-type: "{{ node_type }}"
```

Comment:
Sure, that is a much better name. Would you prefer subhost-type or just sub-type?